<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ISetupDialog</class>
 <widget class="QDialog" name="ISetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>&amp;General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frameGeneral">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="labelPoll">
            <property name="text">
             <string>Poll</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBoxPoll">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>360</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="checkBoxDockInPanel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Doc&amp;k in panel</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="checkBoxUseSessionManagement">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>&amp;Use session management</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="checkBoxDontCheck">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Don'&amp;t check at startup. </string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxIcons">
         <property name="title">
          <string>Icons:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1,1" columnstretch="0,0,0,0,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelStopped">
            <property name="text">
             <string>Stopped:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QToolButton" name="toolButtonStopped">
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="labelNoMail">
            <property name="text">
             <string>No Mail:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelNoMailbox">
            <property name="text">
             <string>No Mailbox:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QToolButton" name="toolButtonNoMailbox">
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QToolButton" name="toolButtonNoMail">
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QToolButton" name="toolButtonOldMail">
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="labelOldMail">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string>Old Mail:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="labelNewMail">
            <property name="text">
             <string>New Mail:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QToolButton" name="toolButtonNewMail">
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonIconOnly</enum>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>labelNoMail</zorder>
         <zorder>toolButtonStopped</zorder>
         <zorder>labelNoMailbox</zorder>
         <zorder>toolButtonNoMailbox</zorder>
         <zorder>toolButtonNoMail</zorder>
         <zorder>labelStopped</zorder>
         <zorder>toolButtonOldMail</zorder>
         <zorder>labelOldMail</zorder>
         <zorder>labelNewMail</zorder>
         <zorder>toolButtonNewMail</zorder>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMailbox">
      <attribute name="title">
       <string>&amp;Mailbox</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QFrame" name="frameServers">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QListWidget" name="listWidgetServers"/>
          </item>
          <item>
           <widget class="QFrame" name="frameServerToolButtons">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QToolButton" name="toolButtonServerAdd">
               <property name="toolTip">
                <string>Add a mailbox</string>
               </property>
               <property name="text">
                <string>&amp;Add</string>
               </property>
               <property name="icon">
                <iconset resource="../resources.qrc">
                 <normaloff>:/icons/32x32/edit-add.png</normaloff>:/icons/32x32/edit-add.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="arrowType">
                <enum>Qt::NoArrow</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButtonServerDelete">
               <property name="toolTip">
                <string>Delete a mailbox</string>
               </property>
               <property name="text">
                <string>Delete</string>
               </property>
               <property name="icon">
                <iconset resource="../resources.qrc">
                 <normaloff>:/icons/32x32/edit-delete.png</normaloff>:/icons/32x32/edit-delete.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameProtocol">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="3" column="0">
           <widget class="QLabel" name="labelProtocol">
            <property name="text">
             <string>Protocol</string>
            </property>
            <property name="buddy">
             <cstring>comboBoxProtocol</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="comboBoxProtocol"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelServer">
            <property name="text">
             <string>Server</string>
            </property>
            <property name="buddy">
             <cstring>lineEditServer</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEditServer">
            <property name="maxLength">
             <number>256</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelPort">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Port</string>
            </property>
            <property name="buddy">
             <cstring>lineEditPort</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEditPort">
            <property name="maxLength">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="labelUser">
            <property name="text">
             <string>User</string>
            </property>
            <property name="buddy">
             <cstring>lineEditUser</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="lineEditUser">
            <property name="maxLength">
             <number>256</number>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="labelPassword">
            <property name="text">
             <string>Password</string>
            </property>
            <property name="buddy">
             <cstring>lineEditPassword</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="lineEditPassword">
            <property name="maxLength">
             <number>256</number>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="labelIMAPMailbox">
            <property name="text">
             <string>Mailbox</string>
            </property>
            <property name="buddy">
             <cstring>lineEditIMAPMailbox</cstring>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="lineEditIMAPMailbox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maxLength">
             <number>256</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelName">
            <property name="text">
             <string>Name</string>
            </property>
            <property name="buddy">
             <cstring>lineEditName</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditName"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAbout">
      <attribute name="title">
       <string>&amp;About</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QFrame" name="frameHeader">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,1">
          <item>
           <widget class="QLabel" name="labelIcon">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resources.qrc">:/icons/64x64/traybiff.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelBiff">
            <property name="font">
             <font>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;TrayBiff&lt;/span&gt; Version&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="textFormat">
             <enum>Qt::RichText</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelVersion">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameText">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This software is licensed under GPL3 or any later version&lt;/p&gt;&lt;p&gt;© 2024 Ulrich Eckhardt &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBoxPoll</tabstop>
  <tabstop>checkBoxDockInPanel</tabstop>
  <tabstop>checkBoxUseSessionManagement</tabstop>
  <tabstop>checkBoxDontCheck</tabstop>
  <tabstop>toolButtonStopped</tabstop>
  <tabstop>toolButtonNoMailbox</tabstop>
  <tabstop>toolButtonNoMail</tabstop>
  <tabstop>toolButtonOldMail</tabstop>
  <tabstop>toolButtonNewMail</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>listWidgetServers</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>comboBoxProtocol</tabstop>
  <tabstop>lineEditServer</tabstop>
  <tabstop>lineEditPort</tabstop>
  <tabstop>lineEditUser</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>lineEditIMAPMailbox</tabstop>
  <tabstop>toolButtonServerAdd</tabstop>
  <tabstop>toolButtonServerDelete</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ISetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ISetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
